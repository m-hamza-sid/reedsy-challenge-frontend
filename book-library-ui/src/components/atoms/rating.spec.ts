import { mount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import Rating from "./rating.vue";

describe("rating", () => {
  test("should render with rating prop", () => {
    const iRating = "05";
    const wrapper = mount(Rating, { props: { rating: iRating } });
    expect(wrapper.text()).toContain(iRating);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
