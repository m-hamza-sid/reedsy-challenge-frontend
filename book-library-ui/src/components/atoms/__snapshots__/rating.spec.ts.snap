// Vitest Snapshot v1

exports[`author > should render with name prop 1`] = `"<div class=\\"text-left\\">some name</div>"`;

exports[`rating > should render with rating prop 1`] = `"<span> (05 / 10) </span>"`;
